client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content/site-status-column.tsx

```diff
@@ -47,7 +47,8 @@ export default function SiteStatsColumn( { type, rows, metadata, disabled }: Pro
 	);
 
 	const partner = useSelector( getCurrentPartner );
-	const partnerCanIssueLicense = Boolean( partner?.can_issue_licenses );
+	const partnerCanIssueLicense =
+		isEnabled( 'a8c-for-agencies' ) || Boolean( partner?.can_issue_licenses ); // FIXME: get this from state when isEnabled( 'a8c-for-agencies' )
```

Comment:
We have the `isA4AEnabled` selector

Reply:
I think this is correct - everywhere we use `isA4AEnabled` it's because it's already been defined using `isEnabled( 'a8c-for-agencies' )` so this is just a shortcut, a better option if it isn't needed elsewhere on the page.

Reply:
True, I was wrong...